proto/endpoint_server.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+import "forch/proto/shared_constants.proto";
+
+/*
+ * Tunnel endpoint params.
+ */
+ message Endpoint {
+    // IP address of tunnel endpoint.
+    string ip = 1;
+  
+    // UDP port for tunnel.
+    int32 port = 2;
+  
+    // Tunnel key index.
+    int32 vni = 3;
+
+    // Port on t1 switch
+    int32 tap_port = 4;
+  }
+
+/*
+ * Device testing session server endpoint.
+ */
+ service EndpointServer {
+     // Configure Vxlan interface
+    rpc ConfigureInterface(Endpoint) returns (Empty) {}
+    rpc CleanupInterface(Endpoint) returns (Empty) {}
```

Comment:
a pontificate that I wouldn't bother addressing but I need to ruminate... Ideally I think this would be a streaming RPC such that if the client were to die for some reason, there would be an implicit "cleanup" on termination of the connection. Not worth fixing since this isn't code I think we should invest a lot in at this point, but there's likely "garbage collection" issues introduced by this!

Reply:
The stream is already on the session server so we know when a connection is terminated. During the implicit cleanup of these sessions, the interfaces used are explicitly cleaned up. I don't think we need to maintain another stream from session server to the endpoint handler